validator/src/main/java/com/google/daq/mqtt/validator/SequenceValidator.java

```diff
@@ -234,6 +234,18 @@ private static Metadata readDeviceMetadata() {
     }
   }
 
+  protected static String getTimestamp(Date date) {
+    try {
+      if (date == null) {
+        return "null";
+      }
+      String dateString = OBJECT_MAPPER.writeValueAsString(date);
+      return dateString.substring(1, dateString.length() - 1);
```

Comment:
Please add a comment that describes what this output will look like.  Is the first char 'T' ?